kernel/src/hil/radio.rs

```diff
@@ -118,3 +118,73 @@ pub trait RadioData<'a> {
         frame_len: usize,
     ) -> Result<(), (ErrorCode, &'static mut [u8])>;
 }
+
+#[derive(PartialEq, Debug, Copy, Clone)]
+pub enum RadioChannel {
+    DataChannel11 = 5,
+    DataChannel12 = 10,
+    DataChannel13 = 15,
+    DataChannel14 = 20,
+    DataChannel15 = 25,
+    DataChannel16 = 30,
+    DataChannel17 = 35,
+    DataChannel18 = 40,
+    DataChannel19 = 45,
+    DataChannel20 = 50,
+    DataChannel21 = 55,
+    DataChannel22 = 60,
+    DataChannel23 = 65,
+    DataChannel24 = 70,
+    DataChannel25 = 75,
+    DataChannel26 = 80,
+}
+
+impl RadioChannel {
+    /// Returns the u8 value of the channel, which is the IEEE 802.15.4 channel
+    pub fn get_channel_index(&self) -> u8 {
```

Comment:
```suggestion
    pub fn get_channel_number(&self) -> u8 {
```

I think that is more common.